docs/content/contribute/define-meeting/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: OSCAL DEFINE Meeting
+description: Details about the OSCAL DEFINE Meeting
+layout: devlunch
+aliases:
+- /contribute/model-review/
```

Comment:
I suggest adding, if it makes sense, the 'oscal-research' and 'oscal-define' aliases

Reply:
Seeing as this is for the `/define-meeting` I didn't add the `/oscal-define` alias because it would be redundant for OSCAL and both of those aliases suggest we will put more generic information there, right? I thought inevitably information, except the meeting invitations will, go elsewhere on the site.

I only added these aliases was to not break those sites linking externally to the old meetings and make sure people still arrive at them.

Reply:
Actually I will be adding these, disregard my previous comment. This makes sense.